src/cdk/menu/menu.md

```diff
@@ -108,8 +108,8 @@ element you want to serve as the content for your pop-up menu. Then wrap the con
 will allow the trigger to show and hide the menu content as needed.
 
 <!-- example({
-  "example": "cdk-menu-standalone-menu",
-  "file": "cdk-menu-standalone-menu-example.html"
+  "example": "cdk-menu-standalone-trigger",
```

Comment:
@mmalerba I made a guess that the correct name should be `-trigger`, because the example has `@title Menu with Standalone Trigger.`. That being said, it seems a little weird that the same file is linking this example in 3 different places. Maybe an example got deleted by accident at some point?

Reply:
I think it was originally called `...-standalone-trigger` and I renamed it to `...-standalone-menu`. I must've missed updating a couple places. Could you go with the `...-standalone-menu` naming instead?

Reply:
Done